docker/django/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.4
+
+ENV PYTHONUNBUFFERED 0
+
+RUN apt-get update \
+    && apt-get install -y gdal-bin python-gdal python3-gdal
+
+# Let's install git packages from requirements.txt here
+RUN mkdir -p /src
+
+COPY requirements.txt /src/
+
+RUN pip install --upgrade pip \
+    && pip install --upgrade typing \
+    && pip install pip-tools \
+    && pip install -r /src/requirements.txt
```

Comment:
Use `pip install --src /usr/src -r requirements.txt` to install in /usr/src. Nifty! Saves three lines!

Reply:
nifty indeed!